source/includes/steps-realm-cli-generate-api-key.yaml

```
@@ -22,12 +22,17 @@ content: |
      :alt: Click Access Manager
      :lightbox:
 
-  Clicking this button will navigate you to the "Create API Key" screen.
-  Set a description for your key and a project permission. The various
-  project permissions are described on the right hand side of the screen
-  and can be changed at a later time. Copy the public key to use later
-  in order to log in. Click :guilabel:`next` to continue configuring your
-  key details.
+  Clicking this button will navigate you to the "Create API Key" screen. Set a
```

```suggestion
  Clicking this button navigates you to the "Create API Key" screen. Set a
```
Passive -> active voice, take it or leave it.

---

I'll take it, but nit: both are active voice. The thing to avoid here is future tense, which muddies the sentence with modifying verbs.

---

Good point, not passive voice. Glad we're cleaning up the sentence.